cpp/libcudf_kafka/include/cudf_kafka/kafka_consumer.hpp

```diff
@@ -48,15 +52,27 @@ class kafka_consumer : public cudf::io::datasource {
    *
    * @param configs key/value pairs of librdkafka configurations that will be
    *                passed to the librdkafka client
+   * @param python_callable `python_callable_type` pointer to a Python functools.partial object
```

Comment:
It doesn't need to be a partial, right? Any callable will do (i.e. anything implementing `__call__`).

Reply:
Honestly I am not sure on this. We are trying to maintain 100% compatibility with python-confluent-kafka and that specifies a functools.partial so just wanted to make sure it was the same